### PR TITLE
Create adventOfCode062022.hs

### PR DESCRIPTION
**Pull request by Anonymus-2022**
"main" was an help of my University. I didn't pay attention to the runtime.
I'm a student and I'm happy about any improvements to the code

### CODE REVIEW

**Comment by janiaz708** [comment]
hi

**Comment by Resultsvary777** [comment]
Hi

File: adventOfCode062022.hs [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    case args of
+        [] -> noFileFound ""
+        (fileName:_) -> do
+            exists <- doesFileExist fileName
+            if exists
+                then do
+                    content <- readFile fileName
+                    doStuff content
+                else noFileFound fileName
+
```
Comment (janiaz708) [inline adventOfCode062022.hs:60]: dostuff   content

**Comment by liedlgruber** [comment]
Anonymus-2022:patch-1

**Review by liedlgruber — APPROVED** [review]

**Review by liedlgruber — COMMENTED** [review]
adventOfCode062022.hs

**Comment by liedlgruber** [comment]
https://github.com/github/codespaces-blank/pull/3#pullrequestreview-1248676351

**Review by liedlgruber — CHANGES_REQUESTED** [review]
vierGleiche (a:b:c:d:ds) = if ((length (a:b:c:d:ds))<4)
    then[False]
    else if ((a==b) || (a==c) || (a==d) || (b==c) || (b==d) || (c==d))
        then False : (vierGleiche (b:c:d:ds))
        else [True]


--the length of the datastream until the start-of-packet

File: adventOfCode062022.hs (hunk 1 above)
Comment (liedlgruber) [reply adventOfCode062022.hs:60]: adventOfCode062022.hs

File: adventOfCode062022.hs [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+doStuff :: String -> IO ()
+doStuff content = do
+    putStrLn (solution6 content)
+    putStrLn ""
```
Comment (liedlgruber) [inline adventOfCode062022.hs:65]: doStuff :: String -> IO ()
doStuff content = do
    putStrLn (solution6 content)putStrLn (solution62 content)
    putStrLn ""

**Review by liedlgruber — APPROVED** [review]

**Review by liedlgruber — APPROVED** [review]
adventOfCode062022.hs

**Review by liedlgruber — CHANGES_REQUESTED** [review]
adventOfCode062022.hs

**Comment by Diamondbrandblue** [comment]
Im happy to work with it

**Review by Diamondbrandblue — APPROVED** [review]
Im working on figuring out differential nonsplit

**Review by Domi88sss — APPROVED** [review]

**Review by Domi88sss — APPROVED** [review]

**Review by Domi88sss — APPROVED** [review]

**Comment by macrombilux** [comment]
> hi

File: adventOfCode062022.hs (hunk 1 above)
Comment (macrombilux) [reply adventOfCode062022.hs:60]: 🚀

Comment (Charisma-Mua) [reply adventOfCode062022.hs:60]: That is cool. Please can I get a project?

**Review by Dioliode — APPROVED** [review]
DioX

**Pull request closed without merging**